src/main/kotlin/com/wg2/storefront/products/ProductService.kt

```diff
@@ -38,6 +44,14 @@ object ProductService {
         subtitle = subtitle,
         description = description,
         productUrl = productUrl,
+        icon = icon.toImage(),
+        banner = banner.toImage(),
+        images = imagesList.map { it.toImage() }
+//        images = (1..8).map {
+//            Image("https://picsum.photos/1024/512?random=$it")
+//        }
```

Comment:
Looks like this should be removed?

Reply:
It was helpful to me to test with more than just 0..1 picture(s), figured it could be helpful to someone else too.

Reply:
Err, why does `imagesList` only have one image then? 🤔

Reply:
We get that from the API, i.e. that's the amount of pictures on these particular products, added in devportal. If that's the question?

Reply:
Yes, I'm just thinking maybe there should be more than one image added if that's what we want for testing. So you don't have to hardcode fake data.

Reply:
Not a blocker, I just think that would be a nicer solution. Alternatively adding a comment over the commented-out code to explain why it's there.

Reply:
[67b1287](https://github.com/working-group-two/storefront-example/pull/2/commits/67b1287ba1b1fd6f81e8802baf9a2174d95acd1f)